extensions/amp-onetap-google/validator-amp-onetap-google.protoascii

```diff
@@ -0,0 +1,45 @@
+#	
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.	
+#	
+# Licensed under the Apache License, Version 2.0 (the "License");	
+# you may not use this file except in compliance with the License.	
+# You may obtain a copy of the License at	
+#	
+#      http://www.apache.org/licenses/LICENSE-2.0	
+#	
+# Unless required by applicable law or agreed to in writing, software	
+# distributed under the License is distributed on an "AS-IS" BASIS,	
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.	
+# See the License for the specific language governing permissions and	
+# limitations under the license.	
+#	
+
+tags: {  # amp-onetap-google
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-onetap-google"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"	
+}
+
+tags: {  # <amp-onetap-google>
+  html_format: AMP
+  tag_name: "AMP-ONETAP-GOOGLE"
+  requires_extension: "amp-onetap-google"
+  unique: true
```

Comment:
you have included unique but don't test for it, which i'm fine with, just wanting to state I still think we could put these test files into one test file, but it's a very very small nit =D

Reply:
Ah, cool! Done: one file, three cases.